docs/src/components/VersionDropdown/index.js

```diff
@@ -23,7 +24,7 @@ export const getSelectedVersionPath = (
   selectedVersionDocsPath = selectedVersionDocsPath.endsWith("/")
     ? selectedVersionDocsPath.substring(0, selectedVersionDocsPath.length - 1)
     : selectedVersionDocsPath;
-  return `${selectedVersionDocsPath}${slugWithoutVersion}`;
+  return `${selectedVersionDocsPath}${followPath ? slugWithoutVersion : ''}`;
```

Comment:
This will make `/docs/1.1.1/concepts/cas-engines` to `/docs/1.1.2/` while changing the version
let me know if this is okay!. If all the docs are going to be having the same slug then it will be okay to not change any code here.
cc: @IsAmrish @satyapriyamishra222

Reply:
> If all the docs are going to be having the same slug then it will be okay to not change any code here.

yes, if somehow it is different ( it sometime happens ), we can redirect the page to root of that version.

Reply:
rest code looks good 👍🏻

Reply:
Working on it… Seems to be some change in the version structures

Reply:
Seems fine. Let's redirect to the root if not available.

Reply:
this is already done.